layouts/error.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="layout-default">
+      <div class="pt-4 mt-12">
+        <center>
+            <div class="mt-12 pt-12">
+                <h1 v-if="error.statusCode === 404">This page could not be found</h1>
+                <h1 v-else>There was an unknown error</h1>
+            </div>
+        </center>
+      </div>
+      <div tyle="padding-bottom: 200px;" />
```

Comment:
@CryptoBender21 
As we have tailwindcss in place, I'd say we should use tailwindcss instead of editing the style attribute with magic numbers.

Reply:
Agreed, was taken from a different template for the sake of consistency